src/CONTRIBUTORS.js

```diff
@@ -457,3 +457,14 @@ export const Khadaj = {
     link: 'https://worldofwarcraft.com/en-us/character/firetree/khadaj',
   }],
 };
+export const fel1ne = {
+  nickname: 'fel1ne',
+  github: 'fel1n3',
+  discord: 'Dr. fel1ne#5614',
+  avatar: require('./images/fel1ne-avatar.png'),
```

Comment:
You need to add your avatar to version control to Git otherwise the server won't have it